app/services/karma_calculator.rb

```diff
@@ -43,27 +43,11 @@ def number_hangouts_started_with_more_than_one_participant
     user.number_hangouts_started_with_more_than_one_participant
   end
 
-  def hangouts_attended_with_more_than_one_participant
```

Comment:
should we be moving this code into a rake task so that when/if participation data gets fed in again we can run it as a separate one off task to update the csv file?

Reply:
you make a strong argument :-) as @nisevi knows I'm a big fan of keeping things around :-) but I guess there's always git history, and you're right, it's a very inefficient implementation and if we do get something new set up we should plan and implement that separately ... okay let's merge and get you working on the jitsi stuff - that's a huge thing for us that could really streamline the community ...